src/tekhsi/tek_hsi_connect.py

```diff
@@ -59,7 +62,7 @@ def __init__(self, url: str, activesymbols=None, callback=None, data_filter=None
            callback (function, optional): Callback function to handle incoming data.
            data_filter (function, optional): Filter function to apply to incoming data.
         """
-        self.prevheaders = []
+        self.previous_headers = []
```

Comment:
@tekperson, I made this change so that the attribute was more readable. Is that acceptable?